Library/Formula/plan9port.rb

```diff
@@ -4,8 +4,18 @@ class Plan9port < Formula
   url "https://plan9port.googlecode.com/files/plan9port-20140306.tgz"
   sha256 "cbb826cde693abdaa2051c49e7ebf75119bf2a4791fe3b3229f1ac36a408eaeb"
 
+  depends_on :x11 => :recommended
```

Comment:
We don't allow new recommended or mandatory X11 dependencies in the core anymore because X11 is generally a bad UX and forcing people to go install it to use formulae is not something we want to do. We recommend `=> :optional` here or if it's absolutely necessary move it to the `homebrew/x11` tap.